docs/glossary.rst

```diff
@@ -86,7 +97,7 @@ community discussions.
   RTS
     Real Time Strategy
 
-    This is a type of game where gameplay occurs all the time. In the Longturn community, this term is also
+    This is a type of game where game play occurs all the time. In the Longturn community, this term is also
```

Comment:
"gameplay" is a single word, this change is unnecessary

Reply:
I always get stuck on that. Thanks.